libc/src/math/amdgpu/fmaximum.cpp

```diff
@@ -0,0 +1,22 @@
+//===-- Implementation of the fmaximum function for GPU
+//-----------------------===//
```

Comment:
Make it fit in 1 line, and exact 80-char.

Reply:
same with other files in the directory `amdgpu`? or everywhere else?

Reply:
Everywhere, the headers should all look like the other ones.

Reply:
ok thanks